server/src/main/java/org/spine3/server/reflect/EventHandlerMethod.java

```diff
@@ -117,25 +111,24 @@ public EventHandlerMethod create(Method method) {
 
         private enum Singleton {
             INSTANCE;
-            @SuppressWarnings("NonSerializableFieldInSerializableClass")
-            private final Factory value = new Factory();
+            @SuppressWarnings({"NonSerializableFieldInSerializableClass", "UnnecessarilyQualifiedInnerClassAccess"})
```

Comment:
IDEA complains about that (shows compilation errors though Gradle build passes). I think this is because the base factory class name is the same. 
Also, should I demote `UnnecessarilyQualifiedInnerClassAccess` warning only, or `NonSerializableFieldInSerializableClass` also?